views/blocks/thankyou.tpl

```diff
@@ -7,9 +7,53 @@
 *
 *}]
 
+[{assign var="paymentInAdvanceInfo" value=$oView->getPaymentInAdvanceInfo()}]
+
+[{if $paymentInAdvanceInfo && $paymentInAdvanceInfo->sIban}]
+    <style>
+        #pia-payment-info {
+            margin-bottom: 20px;
+        }
+        #pia-payment-info, #pia-payment-info td {
```

Comment:
The second statement should be on a new line.
```suggestion
        #pia-payment-info,
        #pia-payment-info td {
```

Reply:
👍